src/schemas/authSchemas.ts

```diff
@@ -1,6 +1,15 @@
 import * as yup from 'yup';
```

Comment:
Same here, since you have already used yup there is option to do errors handling inside each field (picking up errors from useForm) or you can use your schema and pass it up to useForm: https://react-hook-form.com/get-started#SchemaValidation.